packages/hebao_v1/contracts/modules/dapps/PoolTogetherModule.sol

```diff
@@ -29,15 +29,16 @@ contract PoolTogetherModule is SecurityModule
     {
     }
 
-    function extractMetaTxSigners(
-        address /*wallet*/,
-        bytes4  /*method*/,
-        bytes   memory /*data*/
+    function verifySigners(
+        address   /*wallet*/,
+        bytes4    /*method*/,
+        bytes     memory /*data*/,
+        address[] memory /*signers*/
         )
         internal
         view
         override
-        returns (address[] memory signers)
+        returns (bool)
```

Comment:
Do we need to return  a default value false?

Reply:
By not returning anything the return value will always be false I think.
But these are just placeholders and the implementation still needs to be done so I don't think it matters either way.